CMake/sitkLanguageOptions.cmake

```diff
@@ -42,13 +58,12 @@ endmacro()
 #-----------------------------------------------------------
 # Lua
 
-set_QUIET( WRAP_LUA )
-find_package ( Lua ${_QUIET} )
+set_DEFAULT_QUIET( WRAP_LUA )
 
-if ( LUA_FOUND )
-  set( WRAP_LUA_DEFAULT ${WRAP_DEFAULT} )
-else()
-  set( WRAP_LUA_DEFAULT OFF )
+if( NOT DEFINED WRAP_LUA_DEFAULT OR WRAP_LUA )
```

Comment:
Instead of checking variable `WRAP_<languageName>` and `WRAP_<languageName>`, I suggest to have a variable named `SITK_<languageName>_DO_FIND_PACKAGE` (or similar), then code  would simply be:
```
if( SITK_<languageName>_DO_FIND_PACKAGE )
```

the macro `set_DEFAULT_QUIET` could then be renamed too `sitkShouldDoFindPackageCall(<languageName>)` and set the two variables:
* ` SITK_<languageName>_DO_FIND_PACKAGE`
* ` SITK_<languageName>_FIND_PACKAGE_EXTRA_ARGS`